python/cugraph/dask/common/input_utils.py

```diff
@@ -196,6 +197,10 @@ def _workers_to_parts(futures):
         if w_to_p_map[w] is None:
             w_to_p_map[w] = []
         w_to_p_map[w].append(p)
+    _w_to_p_map = w_to_p_map.copy()
+    for w, p in _w_to_p_map.items():
+        if p is None:
+            del w_to_p_map[w]
```

Comment:
I don't know if this is any faster, but it prevents having to copy the entire dictionary:
```suggestion
    keys_to_delete = [w for (w,p) in w_to_p_map.items() if p is None]
    for k in keys_to_delete:
        del w_to_p_map[k]
```

Reply:
A list comprehension to get the keys that have None values looks better and now I am only iterating over the keys to delete.